packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/SfoxCheckout/Content/OrderCheckout.js

```diff
@@ -30,19 +34,9 @@ const OrderCheckout = ({ quoteR, account, onFetchQuote, reason, finishAccountSet
     ? <FormattedMessage id='buy.sfoxcheckout.inputmethod.title.buywith' defaultMessage='I will pay with' />
     : <FormattedMessage id='buy.sfoxcheckout.outputmethod.title.sellwith' defaultMessage='I will receive funds into' />
 
-  const limitsHelper = (quoteR, limits) => {
-    if (quoteR.error) return true
-    return quoteR.map(q => {
-      if (q.baseCurrency === 'USD') return +q.baseAmount > limits.max || +q.baseAmount < limits.min || +q.quoteAmount > limits.effectiveMax
-      if (q.baseCurrency === 'BTC') return +q.quoteAmount > limits.max || +q.quoteAmount < limits.min || +q.baseAmount > limits.effectiveMax
-    }).data
-  }
-
   const submitButtonHelper = () => (
     reason.indexOf('has_remaining') > -1
-      ? <StepTransition next Component={Button} style={spacing('mt-45')} nature='primary' fullwidth disabled={!Remote.Success.is(quoteR) || limitsHelper(quoteR, limits) || !buttonStatus}>
```

Comment:
Previously, the button for going to the next step after fetching a quote was located far away from where a lot of the data is that it needs to be operational. It's now located in the `QuoteInputTemplate`, which cuts down on some confusion.